Solutions/DNS Domain Solution/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "publisherId": "azuresentinel",
+    "offerId": "azure-sentinel-solution-DNS-preview",
```

Comment:
@vakohl, can you please remove -preview from offer id, if the the offer is still not created, thanks.

Reply:
done